<launch>


    <node pkg="ros_tutorial" type="controller.py" name="controller" output="screen">
        <param name="linear_speed" value="0.2" />
        <param name="angular_speed" value="-0.2"/>
        <param name="goal_angle" value="90"/>
        <param name="stop_distance" value="1"/>
        <param name="epsilon" value="0.01"/>
        
    </node>

    <node pkg="ros_tutorial" type="monitor.py" name="monitor"></node>

    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_funky_maze.launch">
        <arg name="x_pos" value="-9.0"/>
        <arg name="y_pos" value="2"/>
        <arg name="z_pos" value="0.0"/>
        <arg name="yaw" value="1.57075"/>
    </include>

    <include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>

   
</launch>
